params/protocol_params.go

```diff
@@ -165,6 +168,8 @@ const (
 	BlobTxBlobGaspriceUpdateFraction = 3338477 // Controls the maximum rate of change for blob gas price
 
 	BlobTxTargetBlobGasPerBlock = 3 * BlobTxBlobGasPerBlob // Target consumable blob gas for data blobs per block (for 1559-like pricing)
+
+	L1SloadMaxNumStorageSlots = 5 // Max number of storage slots requested in L1Sload precompile
```

Comment:
Why not move this to line 160 (together with the other L1Sload params)?

Reply:
It looks like that block of code is commented as `// Precompiled contract gas prices`, so I thought it's not suitable there